api/policy.go

```diff
@@ -32,13 +41,17 @@ const (
 	BackupPlanPolicyScheduleDaily BackupPlanPolicySchedule = "DAILY"
 	// BackupPlanPolicyScheduleWeekly is WEEKLY backup plan policy value.
 	BackupPlanPolicyScheduleWeekly BackupPlanPolicySchedule = "WEEKLY"
+
+	// WindowUnset is ANYTIME allowed window cron.
+	WindowCronUnset WindowCron = ""
```

Comment:
Looks like you still plan to use cron? Could you elaborate?

Personally, I am not convinced about this approach since cron is more cryptic.  How to support range, how to support multi range?

Also, have you thought about how to expose this from the UI (let user specify a cron probably is not ideal since it's not friendly)

Reply:
I go cron here for the following reasons:
1. cron has been tested by time, and there are many libraries to support cron in many languages
2. the workload for defining our own struct to support this function is simply too large. Since there is a mature solution, why not just use it?

As for 'How to support range with cron': my thought is to allow user config multiple crons, and we will try to take the union of this crons.

For the UI, I plan to let user specify this with a calendar widget, and there should have some libraries that can do the conversion from unix range to cron.

Reply:
cron is a decent and standard format for computer consumption, but not so much for humans. We need to think about developer experience. One is how to specify this to use this feature, another is the debuggability and observability, cron is not a developer friendly.

By looking at this : * 6 * * * ,  which percentage of developers do you think have enough confidence to understand what does this mean without checking the reference?

Not to mention, we are also going to support ranges and also multiple ranges.

>> the workload for defining our own struct to support this function is simply too large.

Em, maybe not that large if we can scope it properly since we are not going to re-invent the cron, but just to serve our own purpose.